src/config/dynamic-theme-fixes.config

```diff
@@ -1614,6 +1614,12 @@ html:not(.style-scope) {
     --yt-spec-general-background-b: ${#f1f1f1} !important;
     --yt-spec-general-background-c: ${#e9e9e9} !important;
 }
+.ytp-volume-slider-handle {
+    background-color: rgba(255, 255, 255, 0.9);
```

Comment:
Please check how it looks in Light mode, if it is not good, try using `${source-color}` pattern, that will be inverted and brightness/contrast will be applied. Also you should use `!important` keyword to prevent overrides by other style sheets.

So, it could look like:
`background-color: ${white} !important;`

Reply:
@alexanderby, is "255,255,255" not superior to "white"? It is more consistent, and must be converted to RGB anyway, so it requires less interpretation.